massgov/mayflower
# DP-11737  downgrade font weight

## Description
- Updating font weight from 800 -> 700 in the event-teaser and the publish-state areas

## Related Issue / Ticket

- JIRA: DP-11737

## Steps to Test
- View the event-teaser-with-graphic molecule. Check that the font size of the larger numbers is set to 700 weight: https://mayflower.digital.mass.gov/b/feature/DP-11737--downgrade-font-weight/patterns/02-molecules-event-teaser-with-graphic/02-molecules-event-teaser-with-graphic.html?1556627240051
- View the published-state atom. Check that the font size of the text is 700 weight: https://mayflower.digital.mass.gov/b/feature/DP-11737--downgrade-font-weight/patterns/01-atoms-11-text-publish-state/01-atoms-11-text-publish-state.html?1556626071086
